modules/installation/pages/install-intro.adoc

```diff
@@ -21,11 +17,8 @@ Because {productname} allows you to manage large numbers of systems and automati
 ifeval::[{productname} == "SUSE Manager"]
 
 From {slsa}{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy are available as base products, and can be installed with the {slsa} Unified Installer.
-This is the simplest method of installation, and the primary method described in this book.
+This is the default method of installation, and the primary method described in this book.
```

Comment:
```suggestion
This is the default method of installation.
```
The way the Install Guide is organised now it doesn't really have a "primary" method.

Reply:
yes, makes sense.